compiler/core/src/core/config.re

```diff
@@ -270,6 +285,7 @@ let load =
       ++ "'. A workspace must contain a `lona.json` file.",
     )
   | Some(workspacePath) =>
+    Log.warn("Running compiler from: " ++ dirname);
```

Comment:
shall this be using some kind of debug helper instead of always printing it? Something like https://github.com/visionmedia/debug#readme (not sure what's there for Reason)

Reply:
Yeah, that's a good idea